client/src/components/recipes/RecipesShow.svelte

```diff
@@ -0,0 +1,96 @@
+<script>
```

Comment:
better name for this component would be `RecipeView` without the plural `Recipes` because you are displaying only one recipe